tests/unit/storage/data/external/ExternalClients.js

```diff
@@ -48,6 +48,16 @@ const backendClients = [
 ];
 const log = new DummyRequestLogger();
 
+class DummyBucket {
```

Comment:
I think we have other dummy classes around here but a sinon.spy could also work, I believe I added it to arsenal not too long ago.

Reply:
Yep, `sinon` sounds good here!